src/CabanaPD_Solver.hpp

```diff
@@ -462,7 +556,7 @@ class SolverFracture
                           neigh_iter_tag{} );
 
             // Add boundary condition.
-            boundary_condition.apply( exec_space{}, *particles );
+            boundary_condition.apply( exec_space{}, *particles, step * dt );
```

Comment:
I am wondering why here we do not distinguish between:

if ( !boundary_condition.forceUpdate() )

and 

if ( boundary_condition.forceUpdate() )

like in lines 518-524

Reply:
Fixed